scanner/src/lib/dataproducts/Battery_ManufacturingDataSheet_v0.1.svelte

```diff
@@ -0,0 +1,251 @@
+  <DataRow label="Manufacturing date" value={data.manufacturingDate} />
+  <DataRow label="Weight" value={formatNumber(data.weight, "kg")} />
+  <DataRow label="Capacity" value={formatNumber(data.capacity, "Ah")} />
+  <DataRow label="Power" value={formatNumber(data.power, "W")} />
+  <DataRow label="Cell type" value={data.cellType} />
+  <DataRow label="Resistance" value={formatNumber(data.resistance, "Ω")} />
+  <DataRow label="Extinguishing agents" value={data.extinguishingAgents.join(", ")} />
+  <DataRow label="Warranty valid until" value={data.warranty} />
+  <div class="divider" />
+  <div class="title no-bottom-margin">Manufacturer information</div>
+  <div class="subtitle">The details of the battery manufacturer</div>
+  <DataRow label="Name" value={data.manufacturerInformation.name} />
+  <DataRow label="Street name" value={data.manufacturerInformation.streetName} />
+  <DataRow label="Postal code" value={data.manufacturerInformation.postalCode} />
+  <DataRow label="City" value={data.manufacturerInformation.city} />
+  <DataRow label="Country" value={countryListAlpha3[data.manufacturerInformation.country]} />
+  <DataRow label="Website" column link value={data.manufacturerInformation.website} />
+  <DataRow label="Email" column value={data.manufacturerInformation.email} />
+  <div class="divider" />
+  <div class="title no-bottom-margin">Manufacturing location</div>
+  <div class="subtitle">The details of the location of the battery manufacturing plant</div>
+  <DataRow label="City" value={data.manufacturingLocation.city} />
+  <DataRow label="Country" value={countryListAlpha3[data.manufacturingLocation.country]} />
+  <div class="divider" />
+  <div class="title no-bottom-margin">Round trip efficiency</div>
+  <div class="subtitle">The details of the round trip energy efficiency in energy storages</div>
+  <DataRow
+    label="Initial energy efficiency"
+    value={formatNumber(data.roundTripEfficiency.initialEnergyEfficieny, "%")}
+  />
+  <DataRow
+    label="Degraded energy efficiency"
+    value={formatNumber(data.roundTripEfficiency.degradedEnergyEfficiency, "%")}
+  />
+  <div class="divider" />
+  <div class="title no-bottom-margin">Voltage levels</div>
+  <div class="subtitle">The details of the voltage levels of the battery</div>
+  <DataRow label="Nominal" value={`${formatNumber(data.voltageLevels.nominalVoltage, "V")}`} />
+  <DataRow label="Maximum" value={`${formatNumber(data.voltageLevels.maximumVoltage, "V")}`} />
+  <DataRow label="Minimum" value={`${formatNumber(data.voltageLevels.minimumVoltage, "V")}`} />
+  <div class="divider" />
+  <div class="title no-bottom-margin">Temperature range</div>
+  <div class="subtitle">The details of the acceptable temperature values of the battery</div>
+  <DataRow
+    label="Temperature range"
+    value={`${formatNumber(data.temperatureRange.minimumTemperature, "°C")} to ${formatNumber(
+      data.temperatureRange.maximumTemperature,
+      "°C"
+    )}`}
```

Comment:
```suggestion
    value={`${formatNumber(data.temperatureRange.minimumTemperature, "°C")}
            to
            ${formatNumber(data.temperatureRange.maximumTemperature, "°C")}`}
```
Would be more readable, if the formatters allow for this